ADOPTERS.md

```diff
@@ -0,0 +1,15 @@
+ADOPTERS
+========
+
+PUBLIC REFERENCES
+-----------------
+
+This document lists the organizations that use Quarkus based on public information available in blog posts and videos. 
+If organizations would like get added or removed please make a PR and follow these rules to comply with legal: 
```

Comment:
I'd rephrase slightly:

```
If any organization would like get added or removed please make a pull request, and follow these guidelines:
```

(changed organizations from plural form, and omitting "comply with legal" which sounds scary - it's just guidelines anyway)